easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-2.1.0.eb

```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Victor E. Goitea
+# Email: victorg@bmb.sdu.dk
+
+easyblock = 'Tarball'
+
+name = 'CellRanger-ATAC'
+version = '2.1.0'
+
+homepage = "https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac"
+description = """Cell Ranger ATAC is a set of analysis pipelines that process 
+Chromium Single Cell ATAC data."""
+
+toolchain = SYSTEM
+
+# Download manually from https://support.10xgenomics.com/single-cell-atac/software/downloads/latest
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['3f44ba66ae789c2ab03f9c380a6718ca']
```

Comment:
@VictorGoitea apparently this is not a checksum in the required `SHA256` format.

Reply:
Hi @verdurin, fixed the checksum to sha256 format now. I am sorry about it, the company Cellranger provides instead md5sum in the downloading website.